cl/alerts/tasks.py

```diff
@@ -244,3 +309,76 @@ def send_unsubscription_confirmation(
     html = html_template.render(email_context)
     msg.attach_alternative(html, "text/html")
     msg.send()
+
+
+@app.task()
+def send_docket_to_webhook(
+    d_pk: int,
+    since: datetime,
+    webhook_recipients: list[str],
+) -> None:
+    """POSTS the DocketAlert to the recipients webhook(s)
+
+    :param d_pk: The Docket primary key
+    :param since: Start time for querying the docket entries
+    :webhook_recipients: The list of email recipients to send the webhook to
+    :return: None
+    """
+
+    docket_entries = DocketEntry.objects.filter(
+        date_created__gte=since, docket_id=d_pk
+    )
+    if docket_entries.count() == 0:
+        # No new docket entries.
+        return
+
+    webhooks = Webhook.objects.filter(
+        event_type=WebhookEventType.DOCKET_ALERT,
+        user__email__in=webhook_recipients,
+        enabled=True,
```

Comment:
In my last comment I said not to pass down the webhook_recipients variable, so now we need to tweak this query to get "Enabled webhooks of type DOCKET_ALERT where the user is subscribed to d_pk."

I admit I gave this a bit of thought and couldn't get it quite figured out in my head, but I'm sure you'll nail it.

Reply:
Yeah, it was possible to get the recipients within `send_docket_alert_webhooks`

First, get a list of recipients subscribed to the docket.
```
webhook_recipients = User.objects.filter(
            docket_alerts__docket_id=d_pk,
            docket_alerts__alert_type=DocketAlert.SUBSCRIPTION,
    ).distinct()```
    
Then select the DocketAlert Webhooks from these users.
```
webhooks = Webhook.objects.filter(
        event_type=WebhookEventType.DOCKET_ALERT,
        user__in=webhook_recipients,
        enabled=True,
    )
```

Reply:
I still think it's possible to do it in one query, but I won't ask you to try. Two is great (and actually pretty clear). :)